src/NServiceBus.Core.Tests/NServiceBus.Core.Tests.csproj

```diff
@@ -64,12 +64,6 @@
     <Reference Include="Mono.Cecil.Rocks">
       <HintPath>..\packages\Mono.Cecil.0.9.5.4\lib\net40\Mono.Cecil.Rocks.dll</HintPath>
     </Reference>
-    <Reference Include="NServiceBus.NewCompilerBits">
-      <HintPath>TestDlls\NServiceBus.NewCompilerBits.dll</HintPath>
-    </Reference>
-    <Reference Include="NServiceBus.OldCompilerBits">
-      <HintPath>TestDlls\NServiceBus.OldCompilerBits.dll</HintPath>
-    </Reference>
```

Comment:
I think those were only added for unit tests about the changed behavior of roslyn regarding declaringType of anonymous delegates. 
Since those are removed as well, those dlls shouldn't be used?